src/plugins/gui/providers/paybisProvider.ts

```diff
@@ -75,15 +97,55 @@ const asPaymentMethodPairs = asObject({
   pairs: asArray(asPaymentMethodPair)
 })
 
-const asPaybisPairs = asObject({
+const asPaybisBuyPairs = asObject({
   data: asArray(asPaymentMethodPairs)
 })
 
+const asSellPair = asObject({
+  fromAssetId: asString,
+  to: asArray(asString)
+})
+
+const asSellPaymentMethodPairs = asObject({
+  name: asMaybe(asPaymentMethodId),
+  // displayName: asString,
```

Comment:
Doesn't look like we're using this, is there a plan to incorporate this later?

Reply:
It is being used. Otherwise it would be a lint error

Reply:
It is being used. Otherwise it would be a lint error

Reply:
It is being used. Otherwise it would be a lint error